templates/apps/sumo-aws-apps/security-hub-org/security-hub-org-main.yaml

```diff
@@ -511,185 +492,80 @@ Resources:
-      SourceArn: !Ref SecurityHubEnablerTopic
-
-  SecurityHubEnablerSubscription:
-    Condition: install_aws_security_hub_app
-    Type: AWS::SNS::Subscription
-    Properties:
-      Endpoint: !GetAtt SecurityHubEnablerLambdaHelper.Arn
-      Protocol: lambda
-      TopicArn: !Ref SecurityHubEnablerTopic
+        SecurityHubConfigurationRoleName: !Sub "SHERole-${ParentStackName}"
+        SecurityHubOrgLambdaRoleName: !Select [1, !Split ["/", !Ref SumoLogicHelperRoleARN]]
+        DelegatedAdminAccountId: !Ref SecurityAccountId
+        ManagementAccountId: !Ref AWS::AccountId
+        ParentStackName: !Ref ParentStackName
 
-  ScheduledRule:
-    Condition: install_aws_security_hub_app
-    Type: AWS::Events::Rule
-    Properties:
-      Description: SecurityHubEnablerTrigger
-      ScheduleExpression: !Sub 'rate(${ComplianceFrequency} minutes)'
-      State: 'ENABLED'
-      Targets:
-        - Arn: !GetAtt SecurityHubEnablerLambdaHelper.Arn
-          Id: 'DailyInvite'
-
-  PermissionForEventsToInvokeLambda:
-    Condition: install_aws_security_hub_app
-    Type: AWS::Lambda::Permission
-    Properties:
-      FunctionName: !GetAtt SecurityHubEnablerLambdaHelper.Arn
-      Action: lambda:InvokeFunction
-      Principal: 'events.amazonaws.com'
-      SourceArn: !GetAtt ScheduledRule.Arn
-
-  SecurityHubEnablerLambdaHelper:
-    Condition: install_aws_security_hub_app
-    Metadata:
-      cfn_nag:
-        rules_to_suppress:
-          - id: W58
-            reason: "Lambda functions require permission to write CloudWatch Logs"
-          - id: W89
-            reason: "Lambda functions should be deployed inside a VPC"
-    Type: 'AWS::Serverless::Function'
-    Properties:
-      Handler: securityhub.lambda_handler
-      Runtime: python3.8
-      CodeUri:
-        Bucket: !Sub
-                - "${S3Bucket}"
-                - S3Bucket: !Sub '${QSS3BucketName}-${AWS::Region}'
-        Key: !Sub "${QSS3KeyPrefix}templates/apps/sumo-aws-apps/sumologic-app-utils/sumo_app_utils.zip"
-      MemorySize: 512
-      Timeout: 900
-      Role: !Ref SumoLogicHelperRoleARN
-      ReservedConcurrentExecutions: 2
-      Environment:
-        Variables:    
-          ASSUME_ROLE: !Sub "SHERole-${ParentStackName}"
-          MGMT_ACCOUNT_ID: !Ref SecurityAccountId
-          SNS_TOPIC_ARN: !Ref SecurityHubEnablerTopic
-          ENABLE_CIS_STANDARD: !Sub ${EnableCISStandard}
-          CIS_STANDARD_VERSION: !Sub ${CISStandardVersion}
-          ENABLE_PCI_STANDARD: !Sub ${EnablePCIStandard}
-          PCI_STANDARD_VERSION: !Sub ${PCIStandardVersion}
-          ENABLE_SBP_STANDARD: !Sub ${EnableSBPStandard}
-          SBP_STANDARD_VERSION: !Sub ${SBPStandardVersion}
-          REGIONS_TO_ENABLE: !Sub ${RegionsToEnable}
-          CONTROL_TOWER_REGIONS_ONLY: !Sub ${ControlTowerRegionsOnly}
-          ENABLE_PROWLER_INTEGRATION: !Sub ${EnableProwlerIntegration}
-
-  SecurityHubEnablerLambdaCustomResource:
+  SecurityHubLambdaCustomResource:
```

Comment:
@himanshu219 
I'm getting an error on this resource, it seems like the policy for the IAM Role for this resource needs to be updated to add UpdateOrganizationConfiguration API from Security Hub, please check and fix:
"Received response status [FAILED] from custom resource. Message returned: An error occurred (AccessDeniedException) when calling the UpdateOrganizationConfiguration operation: You don't have permissions to access this resource."

Reply:
@gcasilva Please share the values of parameters: Section4SecurityHubAggregationRegion, Section4SecurityHubLinkedRegions

Reply:
@langmtt Below are the value of the parameters. FYI these regions are also correctly enabled as governed regions in Control Tower.
Section4SecurityHubAggregationRegion: us-east-1
Section4SecurityHubLinkedRegions: us-east-2, us-west-1

Reply:
@himanshu219 Tested successfully the deployment with the fix sent. Can you please confirm if there's any additional documentation changes you'll add to this PR before we close it and merge it to main?
Thanks

Reply:
@gcasilva  nops, there is no doc changes required.